src/helpers/store.js

```diff
@@ -2,6 +2,7 @@ import Vue from "vue";
 import Vuex from "vuex";
 import * as fb from "./firebaseConfig";
 import md5 from "js-md5";
```

Comment:
Are you using `md5` to hash passwords? Note that `md5` is a weak hash function that is susceptible to collisions, if not for storing passwords, then `SHA-256` might be warranted.

If this is for storing passwords, then the hash needs additionally to be salted. Salted password hash should use something like `PBKDF2` with `HMAC-SHA-256`, with at least `100_000` rounds, and random taken from cryptographically-secure pseudorandom number generators with occasional seeding from from true random (e.g. from system entropy sources), something like `ChaCha20` with seeding from `/dev/random`.

Reply:
Don't worry, passwords are handled entirely by firebase, here `md5` is only used to get the email address hash used for gravatar.